content/docs/2.5/scalers/elasticsearch.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "Elasticsearch"
+availability = "v2.5+"
+maintainer = "Community"
+description = "Scale applications based on elasticsearch search template query result."
+layout = "scaler"
```

Comment:
`go_file` is missing here, check other scalers docs please

Reply:
done; this should be added to `archetypes/scaler.md` as well.